cli/p2p_collection_add.go

```diff
@@ -26,8 +26,9 @@ import (
 func MakeP2PCollectionAddCommand(cfg *config.Config) *cobra.Command {
 	var cmd = &cobra.Command{
 		Use:   "add [collectionID]",
-		Short: "Add P2P collections",
-		Long:  `Use this command if you wish to add new P2P collections to the pubsub topics`,
+		Short: "Add p2p collections",
```

Comment:
Question: why p2p over P2P?

Reply:
Because it was changed elsewhere so I preferred having it the same here as well. No strong preference but I do want some consistency.

Reply:
Indeed should keep all consistent, but I am not sure why `p2p` instead of `P2P`

Reply:
To be honest, I changed it to that because that's what the others were changed to without giving it too much thought. But P2P is what is used more broadly in the industry. We should probably revert back to P2P

Reply:
✅